ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -208,3 +290,10 @@ message DamlContractState {
 
 }
 
+// Stored information about a given party.
+message DamlPartyValue {
+  // Id of participant where the party is hosted.
+  string participant_id = 1;
+  // A display name associated with the given party.
+  string display_name = 2;
```

Comment:
It is not that simple as it would seem. Modification of the party record has to be verifiable by the all the committers/validators. For that you need a notion of LET for party allocation. I will a comment about it without addressing it ATM.